regolith/tools.py

```diff
@@ -666,6 +669,53 @@ def dereference_institution(input_record, institutions):
                 [db_inst["departments"]], ["name", "aka"], input_record["department"]
             )
 
+def merge_collections(a, b, target_id):
```

Comment:
This needs a test

Reply:
@scopatz OK, done this should be ready for review again...